src/main/java/model/Students.java

```diff
@@ -1,35 +1,64 @@
 package model;
 
-import java.util.Collections;
+import java.security.PublicKey;
+import java.util.ArrayList;
 import java.util.List;
 
 public class Students {
-
     private final List<Student> students;
 
-    public Students(List<Student> studentRepository) {
-        this.students = studentRepository;
+    public Students(List<Student> students) {
+        this.students = new ArrayList<>(students);
     }
 
-    public boolean isExistStudent(String name) {
+    public AttendanceTime findAttendanceTimeByAttendanceDate(String studentName, AttendanceDate attendanceDate) {
         return students.stream()
-                .anyMatch(student -> student.isSameName(name));
+                .filter(student -> student.isSameName(studentName))
+                .findFirst()
+                .orElseThrow().
+                findAttendanceTimeByAttendanceDate(attendanceDate);
```

Comment:
여기 두 가지 의견이 있어요!

1. orElseThrow() 안에 무슨 예외를 던질지 지정해주면 좋겠어요 :)
말론이 나중에 디버깅할 때 예외를 쉽게 추적하기 어려울 수 있어요. 상황에 대한 피드백을 작성해보죠!

2. `.` 이 잘못 찍혀있는것 같아요 :)
```suggestion
                .orElseThrow()
                .findAttendanceTimeByAttendanceDate(attendanceDate);
```